daiquiri/__init__.py

```diff
@@ -0,0 +1,11 @@
+import logging
```

Comment:
This is the issue; you have named a module `daiquiri`, so when you are importing it, it is importing the local one and not the installed module. Please avoid name clashes like this in future.

What you should do is shift this code to `/app/main/logging_config.py`. Also import the logging from daiquiri into Flask app. You need to experiment a bit on this about how to get it working with native flask logging.

Reply:
> Also import the logging from daiquiri into Flask app. You need to experiment a bit on this about how to get it working with native flask logging.
I didn't understand exactly?